.env-dist

```diff
@@ -4,3 +4,14 @@ ALLOWED_HOSTS=*
 SWITCH_NEWSLETTER_MAINTENANCE_MODE=False
 CSP_DEFAULT_SRC=*.allizom.org
 CONTENT_CARDS_URL=https://www-dev.allizom.org/media/
+WAGTAIL_ENABLE_ADMIN=True
+
+# By default, local builds store CMS-user-uploaded media locally, but you can
```

Comment:
Why would anyone want to have their local instance upload things to the cloud?

Reply:
Github mis-theaded my reply by email but it's:

> steve: For developing/debugging the cloud storage integration, mainly

> pmac: I think that means you, possibly Rob. Doesn't seem worth the confusion in a default file for that use case as most bedrock users will never need that.

Fair enough. The default is local-machine storage only, so that's fine - I think a comment is worthwhile, but I'll clarify that it's for specific development reasons only not day-to-day bedrocking 👍